### PR TITLE
Update jenkins plugins and base image in Dockerfile

### PR DESCRIPTION
**Pull request by raravena80**
Update link to getting started docs

### CODE REVIEW

**Comment by raravena80** [comment]
I guess the main README.md also needs to be updated with the right versions. Let me know if this looks good and I'll update the main README.md too

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM jenkins:2.60.2
+FROM jenkins/jenkins:lts
```
Comment (colin-msphere) [inline Dockerfile]: This should be a specific version, such as `jenkins/jenkins:2.60.3`. This allows repeatable builds and ensures CI is building and packaging the intended Jenkins version and plugins.

**Comment by colin-msphere** [comment]
@mespshere-ci ok to test

**Comment by colin-msphere** [comment]
retest this please

**Comment by colin-msphere** [comment]
retest this please

**Pull request merged**